Spec/Tests/AuthTests.swift

```diff
@@ -4320,11 +4320,13 @@ class AuthTests: XCTestCase {
     func test__002__should_accept_authURL_response_with_timestamp_argument_as_string() throws {
         var originalTokenRequest: ARTTokenRequest!
         let tmpRest = ARTRest(options: AblyTests.commonAppSetup())
+        
+        let channelName = uniqueChannelName()
         waitUntil(timeout: testTimeout) { done in
             let tokenParams = ARTTokenParams()
             tokenParams.clientId = "john"
             tokenParams.capability = """
-            {"chat:*":["publish","subscribe","presence","history"]}
```

Comment:
what does this do `*`? Is it just naming or some special character/option?

Reply:
It works _only_ with an exact channel name, otherwise it gives `Error 40160 - action not permitted`. So I assume that's how original author named this channel.